buddyboss/buddyboss-platform
# Fix - Profile picture looks blurry even when upload a high resolution picture

### All Submissions:


Fixes #1543 

### How to test the changes in this Pull Request:

1. Sharp profile image expected on different places like header, activity head in chrome like firefox.
2. Elementor Dashboard page should load 300 x 300 PX image instead of 150 x 150 PX
3. Notice on profile and group picture uploader page should mention image dimension: https://prnt.sc/1xp5y7z

### Connected Theme PR
https://github.com/buddyboss/buddyboss-theme/pull/1710

### Proof Screenshots or Video

Notice: https://ibb.co/xfRrfK6
**Blurry Image**
Before: https://ibb.co/SvF5X5K
After: https://ibb.co/yy82Mpw

### Other information:
